cmd/server/main.go

```diff
@@ -551,7 +551,6 @@ func redirectTo(to string) func(http.ResponseWriter, *http.Request) {
 
 func (p *Proxy) runAdminServer(o *ProxyRunOptions, server *server.ProxyServer) error {
 	muxHandler := http.NewServeMux()
-	muxHandler.Handle("/metrics", promhttp.Handler())
```

Comment:
Can we add a redirect (status code 301) handler to the new port? We can remove it in a bit (and I know we have not gone to 1.0 yet) but its good to start ingraining backward compatibility concerns.

Reply:
Why not just leave this here for backwards compatibility?

Reply:
I'd be ok with that solution as well. I just don't want the endpoint to disappear without explanation or any attempt to maint compatibility.

Reply:
If we are ever plan to stop serving /metrics on the admin port, then we should 301 to the new destination. If we're ok keeping it in perpetuity then go ahead and leave it. If we think we would rather just maintain it in one place then its probably better to put in the 301 now, before usage goes up.

Reply:
Added the 301 redirect